python/ray/experimental/sgd/pytorch/examples/train_example.py

```diff
@@ -40,6 +40,11 @@ def optimizer_creator(model, config):
     return torch.optim.SGD(model.parameters(), lr=1e-2)
 
 
+def scheduler_creator(optimizer, config):
```

Comment:
i might be off about something here, but should this example also show how to call `update_scheduler`?

Reply:
done, added comments